postgres/datadog_checks/postgres/postgres.py

```diff
@@ -749,3 +751,4 @@ def check(self, instance):
                 self.db.commit()
             except Exception as e:
                 self.log.warning("Unable to commit: %s", e)
+        self._version = None  # We don't want to cache versions between runs to capture minor updates for metadata
```

Comment:
(nit) In this case, maybe we should just pass down the version as method parameter instead of caching it on self._version. Seems easier to understand and less error prone.